cmd/podman/ps.go

```diff
@@ -506,41 +579,51 @@ func getTemplateOutput(containers []*libpod.Container, opts batchcontainer.PsOpt
 		}
 		psOutput = append(psOutput, params)
 	}
-	// Sort the ps entries by created time
-	sort.Sort(psSorted(psOutput))
 
 	return psOutput, nil
 }
 
 // getJSONOutput returns the container info in its raw form
-func getJSONOutput(containers []*libpod.Container, opts batchcontainer.PsOptions) ([]psJSONParams, error) {
+func getAndSortParams(containers []*libpod.Container, opts batchcontainer.PsOptions) (psSorted, error) {
```

Comment:
Should we be returning a psSorted here? Casting back to []psParam is probably better